src/js/services/optimizely.js

```diff
@@ -199,11 +199,15 @@ const isExperimentAlreadyViewed = (orgId, experimentKey) => {
     // Uglify /w browserlist force us to do catch (_)
     return false;
   }
-};
+}
 
 // storeExperimentParticipation stores the experiment variationName in the
 // localstorage
-const storeExperimentParticipation = (orgId, experimentKey, variationName) => {
+export function storeExperimentParticipation(
```

Comment:
same here as the previous two comments, unless if there is a reason for switching back to function :)

Reply:
Same comment as above. I was matching how the most recent tests were done with highlightURLHash, Users and sidebar.

Reply:
I think think you are saying that the standard should be using the arrow function and we should move things towards using that and not move things towards being back to function.